src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
+        System.out.println(new Lottery().getRandomBall());
```

Comment:
Duplication code, use loop for this

Reply:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/f6637d20-c5da-403a-ad41-de1a8ff50c2e)

Reply:
not fixed